README.md

```diff
@@ -342,6 +342,61 @@ simply add `--debug` flag to any command.
 
 [[back to top](#databricks-labs-ucx)]
 
+## Table Migration Workflow
+
+The `migrate-tables` workflow comprises tasks designed to migrate tables from the Hive Metastore to the Unity Catalog. The subsequent diagram illustrates the workflow's tasks and their dependency CLI commands.
+```mermaid
+flowchart TD
+    subgraph CLI
+    create_table_mapping[create-table-mapping] --> create_catalogs_schemas[create-catalogs-schemas]
+    create_uber_principal[create-uber-principal]
+    principal_prefix_access[principal-prefix-access] --> migrate_credentials[migrate-credentials]
+    migrate_credentials --> migrate_locations[migrate-locations]
+    migrate_locations --> create_catalogs_schemas
+    end
+    
+    create_uber_principal --> mt_workflow
+    create_table_mapping --> mt_workflow
+    create_catalogs_schemas --> mt_workflow
+    
+    subgraph mt_workflow[workflow: migrate-tables]
+    dbfs_root_delta_mt_task[migrate_dbfs_root_delta_tables]
+    external_tables_sync_mt_task[migrate_external_tables_sync]
+    end
+```
+
+### Dependency CLI commands
+- [`create-table-mapping`](#create-table-mapping-command) - Create `mapping.csv` which will be used by the workflow to identify the targets catalog, schema, table of the HMS table to be migrated. User should review and update the mapping file accordingly before proceeding with the migration workflow.
+- [`principal-prefix-access`](#principal-prefix-access-command) - Identify all the storages used in the workspace and corresponding Azure Service Principal or IAM role that are used in the workspace. It outputs `azure_storage_account_info.csv` or `uc_roles_access.csv` which will be later used by `migrate-credentials`command to create UC storage credentials. The csv can be edited to control which IAM roles or Azure Service Principals should be used to create UC storage credentials later.
```

Comment:
Convention: whenever you mention another command, make it a link, so that users have a consistent UX.